rke.tf

```diff
@@ -4,21 +4,29 @@
 resource "rke_cluster" "rancher_server" {
   depends_on = [null_resource.wait_for_docker]
 
+  bastion_host {
```

Comment:
is this just an instance or an asg?

Reply:
it just points to the instance we created with bastion module

Reply:
ok